volume_group.go

```diff
@@ -297,3 +299,37 @@ func (c *ClientIMPL) GetVolumeGroupSnapshotByName(ctx context.Context, name stri
 	}
 	return volGroupList[0], err
 }
+
+// ConfigureMetroVolumeGroup configures the volume group provided by id for metro replication using the
+// configuration supplied by config and returns a MetroSessionResponse containing a replication session ID.
+func (c *ClientIMPL) ConfigureMetroVolumeGroup(ctx context.Context, id string, config *MetroConfig) (session MetroSessionResponse, err error) {
+	_, err = c.APIClient().Query(
+		ctx,
+		RequestConfig{
+			Method:   "POST",
+			Endpoint: volumeGroupURL,
+			ID:       id,
+			Action:   actionConfigMetro,
+			Body:     config,
+		},
+		&session)
+
+	return session, WrapErr(err)
+}
+
+// EndMetroVolumeGroup ends a metro configuration from a volume group and keeps both volume groups by default. The local copy
+// will retain its SCSI Identities while the remote volume group members will get new SCSI Identities if kept.
+func (c *ClientIMPL) EndMetroVolumeGroup(ctx context.Context, id string, config *EndMetroVolumeGroupOptions) (resp EmptyResponse, err error) {
```

Comment:
config -> options as param name?

Reply:
yep that makes sense, considering they are optional.